tdrs-backend/tdpservice/parsers/validators.py

```
@@ -332,49 +366,52 @@ def startsWith(substring, error_func=None):
         lambda value: value.startswith(substring),
         lambda value, row_schema, friendly_name, item_num: error_func(substring)
         if error_func
-        else f"{row_schema.record_type}: {value} does not start with {substring}.",
+        else f"{format_error_context(row_schema, friendly_name, item_num)}: {value} does not start with {substring}.",
     )
 
 
 def isNumber():
     """Validate that value can be casted to a number."""
     return make_validator(
-        lambda value: value.isnumeric(),
-        lambda value, row_schema, friendly_name, item_num: f"{row_schema.record_type}: {value} is not a number."
+        lambda value: str(value).strip().isnumeric(),
```

noting for team awareness...it looks like this change was made to suppress error messages associated with leading spaces on fields like county fips code, and zip code. i thought there was another fix to address this in #2951. @jtimpe @elipe17 @reitermb

---

one thing to add - `isNumber` is not consistently used for the `COUNTY_FIPS_CODE` field validators across the different schemas - some use `isInStringRange` and `matches` as well. so this change may not be present in every instance we need it